pom.xml

```diff
@@ -139,6 +139,18 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.springboot</groupId>
+                <artifactId>camel-spring-boot-bom</artifactId>
```

Comment:
Can this go the camel spring boot module?

Reply:
Do you mean org.apache.camel.springboot:spring-boot?    I'm okay with this pointing to any dependency within the camel-spring-boot project that can represent the version ${camel-spring-boot-version}

Reply:
Sorry for the confusion - that makes sense to me!    Moved the dependency to `camel-spring-boot-upgrade-recipes`